### ISSUE TITLE
A trick to avoid caching

### ISSUE THREAD

**Issue by Nakul93** [bombardier-gif/covid-vaccine-booking#487]
So I just found this video on youtube which explains how we can change the format of date to avoid getting cached data. I think it would be great if we could integrate this in our code.

What are your thoughts guys?

https://www.youtube.com/watch?v=hf44_-qqddY

**Comment by warlock2k** [comment]
This is cool, I think we can do this. 

However, since this is already public, do you see a patch coming from cowin for the same soon?

**Comment by stockguy78** [comment]
Why do they want to Cache data for UI ? doesn't make sense at all..

**Comment by 6mxo89sqh** [comment]
I noticed this last night while doing some testing.

**Comment by Nakul93** [comment]
> This is cool, I think we can do this.
> 
> However, since this is already public, do you see a patch coming from cowin for the same soon?

Yeah thats the annoying part they probably might release a patch to fix this

**Comment by Nakul93** [comment]
Even with our current script caching doesn't seem to be a problem if we are using findBy* APIs. Today I tested for almost 3hrs  with 7secs refresh rate and was able to book when slots opened up (it was faster than telegram alerts)

**Comment by 6mxo89sqh** [comment]
Can some one try this out? 
https://github.com/6mxo89sqh/covid-vaccine-booking/tree/date-formats

I stopped getting OTP suddenly so unable to test it end-to-end.

**Comment by Nakul93** [comment]
> Can some one try this out?
> https://github.com/6mxo89sqh/covid-vaccine-booking/tree/date-formats
> 
> I stopped getting OTP suddenly so unable to test it end-to-end.

Seems to work fine. Does this change the date format on every API call?

**Comment by 6mxo89sqh** [comment]
Ya it rotates it.

**Comment by shailesh** [comment]
Have you also raised a PR? I'll test it out.

**Comment by shivaslifzz** [comment]
> > Can some one try this out?
> > https://github.com/6mxo89sqh/covid-vaccine-booking/tree/date-formats
> > I stopped getting OTP suddenly so unable to test it end-to-end.
> 
> Tested it, seems to work fine. Does this change the date format on every API call?

I guess it's failing, which find option you used ??

**Comment by Nakul93** [comment]
@shivaslifzz find by date option for district

**Comment by 6mxo89sqh** [comment]
Well, I don't think this idea is worth raising the PR. Currently, find by is super strong and this calendar is still bad. We are still restricted by 20sessions.

**Comment by Nakul93** [comment]
Agree with @6mxo89sqh for now find by option is perfect. But in future if this was to change then we can have this alternative as a fix.

**Comment by Nakul93** [comment]
> Can some one try this out?
> https://github.com/6mxo89sqh/covid-vaccine-booking/tree/date-formats

have you tried this with public APIs?
